torch/distributed/fsdp/fully_sharded_data_parallel.py

```diff
@@ -1871,6 +1913,13 @@ def _lazy_init(self) -> None:
                     "set yet or should have been set to `False`"
                 )
                 fsdp_module._is_root = False
+                if self.sharding_strategy == ShardingStrategy.HYBRID_SHARD:
+                    assert hasattr(self, '_inter_node_pg')
```

Comment:
Is this for mypy? Both attributes are set in this same method, so I am wondering if these asserts are meaningful.